### ISSUE TITLE
Failed to fake [String]?

### ISSUE THREAD

**Issue by qonn** [APIs-guru/graphql-faker#79]
First of all, thank you for the wonderful tool.

In the latest version 2.0.0-rc.10, I tried faking a basic [String], such as:

My schema:
```graphql
type Query {
  letters: [String] @examples(values: ["a", "b", "c"])
}
```

My query:
```graphql
query {
  letters
}
```

graphql-faker respond to me this:
```json
{
  "errors": [
    {
      "message": "String cannot represent value: [function]",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "letters",
        0
      ]
    },
    {
      "message": "String cannot represent value: [function]",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "letters",
        1
      ]
    }
  ],
  "data": {
    "letters": [
      null,
      null
    ]
  }
}
```

**Comment by IvanGoncharov** [comment]
@qonn Thanks for reporting 👍 
Fixed in `2.0.0-rc.11`, can you please test?

**Comment by qonn** [comment]
Yep, it works now. thank you for the fast fix ❤️
